spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/SpatialConnect.java

```diff
@@ -204,15 +213,19 @@ public Boolean call(SCServiceStatusEvent scServiceStatusEvent) {
     /**
      * If you have an instance of SpatialConnect Server, this is how you would register it.
      * Passing in a remote configuration object will use the info to start the connection to the backend.
-     * @param remoteConfig object that represents http and mqtt endpoints {@link SCRemoteConfig}.
+     *
+     * @param remoteConfig object that contains http and/or mqtt endpoints {@link SCRemoteConfig}.
      */
     public void connectBackend(final SCRemoteConfig remoteConfig) {
-        if (serviceGraph.getNodeById(SCBackendService.serviceId()) == null) {
-            backendService = new SCBackendService(context);
-            backendService.initialize(remoteConfig);
-            addService(backendService);
-            startService(backendService.getServiceId());
+        Log.v(LOG_TAG, "connecting to backend");
+        if (remoteConfig.getAuth().equalsIgnoreCase("exchange")) {
+            backendService = new SCExchangeBackendService(context);
+        } else {
+            backendService = new SCMqttBackendService(context);
```

Comment:
To be consistent we probably should name this `SCSpaconBackendService` since we are using `SCExchangeBackendService`

Reply:
Since the API or spatialconnect-server can be behind the MQTT broker, I think  "MqttBackend" is a more explicit name.  "SpaconServer" may imply spatialconnect-server so I'd prefer to keep it as `SCMqttBackendService`

Reply:
For API it will be `SCApiBackendService`  api or spacon server could move to GRPC in the future.